frontend/components/MainPage/header/LastTransaction.tsx

```diff
@@ -22,32 +24,35 @@ const Loader = styled(Skeleton.Input)`
   }
 `;
 
-const POLLING_INTERVAL = 60 * 1000; // 1 minute
-
 /**
  * Displays the last transaction time and link to the transaction on explorer
  * by agent safe.
  */
+// TODO: loop over all supported chains 
 export const LastTransaction = () => {
   const { isPageLoadedAndOneMinutePassed } = usePageState();
   const { multisigAddress } = useAddress();
+  const { activeStakingProgramMeta } = useStakingProgram();
 
   const [isFetching, setIsFetching] = useState(true);
   const [transaction, setTransaction] = useState<TransactionInfo | null>(null);
 
+  const chainId = activeStakingProgramMeta?.chainId;
+
   const fetchTransaction = useCallback(async () => {
     if (!multisigAddress) return;
+    if (!chainId) return;
 
-    getLatestTransaction(multisigAddress)
+    getLatestTransaction(multisigAddress, chainId)
       .then((tx) => setTransaction(tx))
       .catch((error) =>
         console.error('Failed to get latest transaction', error),
       )
       .finally(() => setIsFetching(false));
-  }, [multisigAddress]);
+  }, [multisigAddress, chainId]);
 
   // Poll for the latest transaction
-  useInterval(() => fetchTransaction(), POLLING_INTERVAL);
+  useInterval(() => fetchTransaction(), ONE_MINUTE_INTERVAL);
```

Comment:
actually better to avoid naming constants based on their value, in case you need to update the value you'd need to rename the constant and all the places where it's used, might be inconvenient